examples/README.md

```diff
@@ -13,17 +13,3 @@
  ![alt tag](./images/docker_for_windows.png)
   
 
-## Try Code2Cloud samples:
```

Comment:
Do we have a list of examples in some readme?

Reply:
The idea about this PR was for the sample name to be self sufficient, when the user goes to the examples directory, they should be identify what each sample does. If the sample name contains the details, do we need a list of examples in this readme? 
https://github.com/Thevakumar-Luheerathan/module-ballerina-c2c/tree/restructure-samples/examples